### PR TITLE
Mmm 206 updated statement import part two refactor suggestion

### PR DESCRIPTION
**Pull request by konantovich**

### CODE REVIEW

File: packages/backend/src/layers/functionality/statement/get-monobank-statement.service.ts [hunk 1]
```diff
@@ -1,131 +1,188 @@
...
+    iteration = 0,
+  ): Promise<MonobankIntegrationResponse> {
+    const standardDelay = this.configService.get('app.monobankRequestDelay');
+    const result = await this.monobankIntegration.getStatement(args);
+    const featureFlags = await this.featureFlagStorage.getFeatureFlags();
+    //Uncomment this for error test
+    // errorTest++;
+    // if (errorTest >= 5) {
+    //   result.data = 'unknown error' as any;
+    // }
```
Comment (dhammma) [inline packages/backend/src/layers/functionality/statement/get-monobank-statement.service.ts:102]: I believe we shouldn't have it here in the final codebase

**Review by dhammma — APPROVED** [review]
Approved with few comments
But, please wait for @Anastasiia-Salenko check

**Review by Anastasiia-Salenko — APPROVED** [review]

File: packages/backend/src/layers/functionality/statement/get-monobank-statement.service.ts [hunk 2]
```diff
@@ -1,131 +1,188 @@
...
-        const statementPart = await this.monobankIntegration.getStatement({
-          accountId: accountList[i].id,
-          token: accountList[i].token.token,
-          from: from[j],
-          to: to[j],
-        });
+  private async handleImportForOneAccount(
+    account: Account,
+    importAttemptId: string,
+    i: number,
```
[the remark this answers is not in the document]
Comment (konantovich) [reply packages/backend/src/layers/functionality/statement/get-monobank-statement.service.ts]: Done

File: packages/backend/src/layers/functionality/statement/get-monobank-statement.service.ts (hunk 1 above)
Comment (konantovich) [reply packages/backend/src/layers/functionality/statement/get-monobank-statement.service.ts]: This code is for the convenience of testing how the import will behave during an unknown error. I deleted it

**Pull request merged**